egs/swbd/s5c/local/chain/tuning/run_tdnn_7g.sh

```diff
@@ -0,0 +1,247 @@
+#!/bin/bash
+
+# 7g is as 7f, but reverberating the training data with room impulse responses
+# which leads to better results
+# This script assumes a mixing of the original training data with its reverberated copy
+# and results in a 2-fold training set. Thus the number of epochs is halved to
+# keep the same training time. The model converges after 2 epochs of training,
+# The WER doesn't change much with more epochs of training.
+# local/chain/compare_wer.sh 7f 7g
+# System                       7f        7g
+# WER on train_dev(tg)      14.46     14.27
+# WER on train_dev(fg)      13.23     13.16
+# WER on eval2000(tg)        17.0      16.3
+# WER on eval2000(fg)        15.4      14.6
+# Final train prob     -0.0882071 -0.123325
+# Final valid prob      -0.107545 -0.131798
+# Final train prob (xent)      -1.26246   -1.6196
+# Final valid prob (xent)      -1.35525  -1.60244
+
+
+set -e
+
+# configs for 'chain'
+affix=
+stage=1
+train_stage=-10
+get_egs_stage=-10
+speed_perturb=true
+dir=exp/chain/tdnn_7g  # Note: _sp will get added to this if $speed_perturb == true.
+decode_iter=
+ivector_dir=exp/nnet3_rvb
+num_data_reps=1        # number of reverberated copies of data to generate
+input_train_set=train_nodup
+
+
+# TDNN options
+# this script uses the new tdnn config generator so it needs a final 0 to reflect that the final layer input has no splicing
+splice_indexes="-1,0,1 -1,0,1 -1,0,1 -3,0,3 -3,0,3 -6,0,6 0"
+# smoothing options
+self_repair_scale=0.00001
+# training options
+num_epochs=2
```

Comment:
Did you get a chance to check the log-likelihood values at the end of training ? Did the training converge, is there no improvement from running the training for few more epochs  ?